### PR TITLE
Added test for scenario where day and month have space after digit and a...

### PR DESCRIPTION
**Pull request by grantrobson**
...mended DateTuple class to accept such input

### CODE REVIEW

File: src/main/scala/uk/gov/hmrc/play/mappers/DateTuple.scala [hunk 1]
```diff
@@ -25,7 +25,10 @@ object DateTuple extends DateTuple
 
 trait DateTuple {
 
-  import DateFields._
+  import uk.gov.hmrc.play.mappers.DateFields
```
Comment (xnejp03) [inline src/main/scala/uk/gov/hmrc/play/mappers/DateTuple.scala]: the original import is nicer.

File: src/main/scala/uk/gov/hmrc/play/mappers/DateTuple.scala [hunk 2]
```diff
@@ -45,7 +48,7 @@ trait DateTuple {
           if (y.length != 4) {
             throw new Exception("Year must be 4 digits")
           }
-          new LocalDate(y.toInt, monthOption.getOrElse(throw new Exception("Month missing")).toInt, dayOption.getOrElse(throw new Exception("Day missing")).toInt)
+          new LocalDate(y.toInt, monthOption.getOrElse(throw new Exception("Month missing")).trim.toInt, dayOption.getOrElse(throw new Exception("Day missing")).trim.toInt)
```
Comment (xnejp03) [inline src/main/scala/uk/gov/hmrc/play/mappers/DateTuple.scala:48]: shouldn't we trim the year too?

Comment (grantrobson) [inline src/main/scala/uk/gov/hmrc/play/mappers/DateTuple.scala:48]: Yes, you're right - datetuple already does, but the test didn't test for this. I've amended

File: src/main/scala/uk/gov/hmrc/play/mappers/DateTuple.scala (hunk 1 above)
Comment (grantrobson) [inline src/main/scala/uk/gov/hmrc/play/mappers/DateTuple.scala]: I agree. Don't know where the imports came from but I've amended.

**Pull request merged**